lib/arctic/vendor/api.rb

```diff
@@ -47,8 +47,9 @@ def create_product(shop_id, product)
         response
       end
 
-      def sync_products(shop_id)
-        request :get, "shops/#{shop_id}/products/sync_products"
+      def ready_for_sync_products(shop_id)
```

Comment:
Given that the rake method is named updated, shouldn't this be ready for update?